nvflare/app_common/np/np_model_locator.py

```diff
@@ -29,18 +29,23 @@
 class NPModelLocator(ModelLocator):
     SERVER_MODEL_NAME = "server"
 
-    def __init__(self, model_dir="models", model_name="server.npy"):
+    def __init__(self, model_dir="models", model_name: Union[str, Dict[str, str]] = "server.npy"):
         """The ModelLocator's job is to find the models to be included for cross site evaluation
         located on server. This NPModelLocator finds and extracts "server" model that is saved during training.
 
         Args:
             model_dir (str): Directory to look for models in. Defaults to "model"
-            model_name (str). Name of the model. Defaults to "server.npy"
+            model_name (dict). Name of the model. Defaults to "server.npy"
```

Comment:
Please update the docstring about "model_name". It can be a str of simple model name, or a dict of model_name => location.

Reply:
updated.